airflow/www/templates/airflow/dags.html

```diff
@@ -41,14 +42,25 @@ <h2>DAGs</h2>
         </form>
       </div>
     </div>
+
+    <div id="owner-boxes">
+      {% for owner in owners %}
```

Comment:
sort owners alphabetically:
```suggestion
      {% for owner in sort(owners) %}
```

Reply:
Why do you need `owners`? Don't you have access to `dag.owners` here anyway?

Reply:
`dag.owners` is just the single owner of a given DAG; we need the sum owners of all DAGs.